### ISSUE TITLE
What is the loss function you used? CrossEntropyLoss is seen default loss function. Is it true?

### ISSUE THREAD

**Issue by osivaz61** [IntelliDAL/Image#1]

**Comment by osivaz61** [comment]
.
